include/flatbuffers/idl.h

```diff
@@ -644,10 +646,12 @@ extern bool GenerateGeneral(const Parser &parser,
 // Generate a schema file from the internal representation, useful after
 // parsing a .proto schema.
 extern std::string GenerateFBS(const Parser &parser,
-                               const std::string &file_name);
+                               const std::string &file_name,
+                               const bool &escape_proto_identifiers);
```

Comment:
No need to pass this variable, it is already in `parser.opts.escape_proto_identifiers`